addons/languages/german/pack/src/main/res/xml/popup_e.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would change the name of this file to suggest it's for `de`. maybe `de_popup_e.xml`?